README.md

```diff
@@ -18,11 +18,40 @@ mbed import mbed-os-example-bootloader
 cd mbed-os-example-bootloader
 ```
 
-## Set up application to be a bootloader
+## Connecting the SD card
+This example is configured to use the **onboard SD card port** for the K64F and the UBLOX_EVK_ODIN_W2.
```

Comment:
"is configured" is passive voice. Could we change this to "This example uses the..."?